workloads/kube-burner/common.sh

```diff
@@ -16,6 +16,16 @@ log() {
   echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") ${@}\033[0m"
 }
 
+collect_pprof() {
+  sleep 50
```

Comment:
why we need this sleep?

Reply:
to let kube-burner initialize and get started with the workload